v1/kustomize/overlay/7.2.0/defaults/configmap.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: securebanking-platform-config
+data:
+
+  # Commented out section below needs to be overwritten in kustomize overlay 
+
+  # BASE_FQDN: value_in_overlay
+  # IG_FQDN: value_in_overlay
+  # MTLS_FQDN: value_in_overlay
+  # IDENTITY_PLATFORM_FQDN: value_in_overlay
+  # USER_OBJECT: value_in_overlay
+  # IDENTITY_DEFAULT_USER_AUTHENTICATION_SERVICE: value_in_overlay
+  # IDENTITY_GOOGLE_SECRET_STORE_NAME: value_in_overlay
+  # IDENTITY_GOOGLE_SECRET_STORE_OAUTH2_CA_CERTS_SECRET_NAME: value_in_overlay
+  # IG_OB_ASPSP_SIGNING_KID: value_in_overlay
+
+  # End of kustomize overlay
+
+  # CDK value: (Cloud Developer's Kit) development identity platform
+  # CDM value: CDM (Cloud Deployment Model)
+  # FIDC value: FIDC (Forgerock Identity Cloud) identity cloud platform
+  ENVIRONMENT_TYPE: FIDC
+  # Name of the RS Service
+  RS_INTERNAL_SVC: securebanking-openbanking-uk-rs
+  # Name of the IG Service
+  IG_INTERNAL_SVC: ig
+  # Name of the RCS Service
+  RCS_API_INTERNAL_SVC: securebanking-openbanking-uk-rcs
+  # Name of the RCS UI Service
+  RCS_UI_INTERNAL_SVC: securebanking-ui-rcs-ui
+  # RCS connection settings for the RS API
+  RS_API_URI: http://securebanking-openbanking-uk-rs:8080
+  # Connection settings for the Consent Repo (hosted by IG)
+  CONSENT_REPO_URI: http://ig:80
+  RCS_CONSENT_RESPONSE_JWT_SIGNINGKEYID: rcs-jwt-signer
+  RCS_CONSENT_RESPONSE_JWT_ISSUER: secure-open-banking-rcs
+  # Realm to be used in Cloud Instance
+  AM_REALM: alpha
+  CERT_ISSUER: null-issuer
+  OB_ASPSP_ORG_ID: 0015800001041REAAY
```

Comment:
Move to the overlay.

This is public information but it is specific to an FR deployment so shouldn't be the default value.

Reply:
Done